py/Boinc/setup_project.py

```diff
@@ -339,6 +340,30 @@ def dir(*dirs):
             shutil.copy(srcdir('html/user/sample_motd.php'), dir('html/user/motd.php'))
         os.system("rm -f "+dir('html/languages/translations/*'))
         install_glob(srcdir('html/languages/translations/*.po'), dir('html/languages/translations/'))
+        try:
+            os.remove(os.path.join(dest_dir, 'release.inc'))
+        except OSError:
+            pass
+        try:
+            s = subprocess.Popen(["git", "rev-parse", "HEAD"], stdout=subprocess.PIPE, stderr=open(os.devnull, 'w'))
+            commit = s.stdout.read()[:7]
+            s = subprocess.Popen(["git", "describe", "--match", "server_release/*", "--tags", "--exact", "HEAD"],
+                                 stdout=subprocess.PIPE, stderr=open(os.devnull, 'w'))
+            version = s.stdout.read().split("/")[-1].strip()
+            content = '''<?php
+global $git_commit;
+global $server_version;
+$git_commit = "{commit}";
+$server_version = "{version}";
```

Comment:
I believe that this should be able to retrieve the 0.9 server release tag.  However, it is not found and the file generated looks like this:

root@boincserver:~/project# cat release.inc 
```
<?php
global $git_commit;
global $server_version;
$git_commit = "3d7281d";
$server_version = "0.9";
```

Also - shouldn't the file end with `?>` to be proper?